.github/workflows/project-automation.yml

```
@@ -0,0 +1,138 @@
+      - opened
+      - reopened
+      - closed
+      - ready_for_review
+      - review_requested
+
+jobs:
+  update-project-status:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Project Status
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)